op-e2e/setup.go

```diff
@@ -273,16 +276,18 @@ func (sys *System) NodeEndpoint(name string) string {
 }
 
 func (sys *System) Close() {
-	postCtx, postCancel := context.WithCancel(context.Background())
-	postCancel() // immediate shutdown, no allowance for idling
-
 	if sys.L2OutputSubmitter != nil {
 		sys.L2OutputSubmitter.Stop()
 	}
 	if sys.BatchSubmitter != nil {
-		_ = sys.BatchSubmitter.Kill()
+		ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+		defer cancel()
+		sys.BatchSubmitter.StopIfRunning(ctx)
 	}
 
+	postCtx, postCancel := context.WithCancel(context.Background())
+	postCancel() // immediate shutdown, no allowance for idling
+
```

Comment:
Why change the batch submitter start/stop handling? `develop` is already up to date here, this diff breaks it